### ISSUE TITLE
terraform module azurerm_static_site only supports sku_size : Free

### ISSUE THREAD

**Issue by juanpabosu** [hashicorp/terraform-provider-azurerm#12610]
### Community Note

* Please vote on this issue by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original issue to help the community and maintainers prioritize this request
* Please do not leave "+1" or "me too" comments, they generate extra noise for issue followers and do not help prioritize the request
* If you are interested in working on this issue or have submitted a pull request, please leave a comment

### Terraform (and AzureRM Provider) Version

AzureRM: 2.67.0
Terraform: 1.0.2

### Affected Resource(s)

* `azurerm_static_site`

### Terraform Configuration Files

```hcl
resource "azurerm_static_site" "static_web_app" {
  name                = var.static_name
  resource_group_name = azurerm_resource_group.rg.name
  location            = var.location_static
  sku_size = "Standard"
}
```
### Debug Output

terraform plan
╷
│ Error: expected sku_size to be one of [Free], got Standard
│
│   with azurerm_static_site.static_web_app,
│   on main.tf line 81, in resource "azurerm_static_site" "static_web_app":
│   81:   sku_size = "Standard"

### Expected Behaviour

Create an azure static web app with Standard Hosting Plan

### Actual Behaviour

Create an azure static web app only with Free Hosting Plan

### Steps to Reproduce

1. `terraform plan`

### References

* https://docs.microsoft.com/en-us/azure/static-web-apps/plans

**Comment by mattthias** [comment]
https://github.com/terraform-providers/terraform-provider-azurerm/pull/12510

**Comment by lukebailey** [comment]
Using azurerm provider 2.78 this works with the following - note we had to set both the tier and size to Standard

```
resource "azurerm_static_site" "static_site" {
  name                           = "somename"
  resource_group_name  = azurerm_resource_group.resource_group.name
  location                        = var.location
  sku_tier                        = "Standard"
  sku_size                        = "Standard"
}
```

**Comment by tombuildsstuff** [comment]
Fixed in #12510
